runtime/tests/tests.rs

```diff
@@ -146,11 +147,40 @@ fn advance_session() {
 	start_session(current_index + 1);
 }
 
+// Starts all sessions up to `era_index` (eg, start_era(2) will start 14 sessions)
 fn start_era(era_index: EraIndex) {
 	start_session((era_index * SessionsPerEra::get()).into());
 	assert_eq!(Staking::current_era(), era_index);
 }
 
+fn current_total_payout(validator_count: Balance) -> Balance {
+	let now = Timestamp::now();
+	let previous_era_start = <crml_staking::CurrentEraStart<Runtime>>::get();
+	let era_duration = now - previous_era_start;
+
+	crml_staking::inflation::compute_total_payout(
```

Comment:
to make this more readable, can we do:
`let (payout, _) = crml_staking::inflation::compute_total_...`
and then return `payout` instead of `.0`?